### PR TITLE
Feature/service setup

### PR DESCRIPTION
**Pull request by temitope777**
** Trello card:** 452

Context
Deploy Postgres

Changes proposed in this pull request
Add postgres resource in aks using terraform-modules
Update Makefile to now run terraform init, plan and apply

Guidance to review
make development terraform-plan  
See the already deployed resources (s189t01-gse-dv-pg)

### CODE REVIEW

**Comment by temitope777** [comment]
[Deploy postgres](https://trello.com/c/RLGjgG3f/479-deploy-postgres)

File: .gitignore [hunk 1]
```diff
@@ -1,47 +1,6 @@
-# See https://help.github.com/articles/ignoring-files for more about ignoring files.
```
Comment (saliceti) [inline .gitignore:1]: The current .gitignore content must be kept. We should only add what's missing

File: Makefile [hunk 2]
```diff
@@ -1,136 +1,102 @@
 TERRAFILE_VERSION=0.8
 ARM_TEMPLATE_TAG=1.1.6
 RG_TAGS={"Product" : "Get Schools Experience"}
-SERVICE_SHORT=gse
 REGION=UK South
-
```
Comment (saliceti) [inline Makefile:6]: The original Makefile commands must be kept as they are in use by the live service

File: Makefile [hunk 3]
```diff
@@ -1,136 +1,102 @@
...
-	echo ""
-
-APPLICATION_SECRETS=SE-SECRETS
-INFRA_SECRETS=SE-INFRA-SECRETS
+SERVICE_NAME=get-schools-experience
+SERVICE_SHORT=gse
+DOCKER_REPOSITORY=ghcr.io/dfe-digital/schools-experience
 
 .PHONY: development
 development:
```
Comment (saliceti) [inline Makefile:10]: Can it be renamed to development_aks so it doesn't clash with the original "development" paas deployment?

File: Makefile [hunk 4]
```diff
@@ -1,136 +1,102 @@
...
+			"tfStorageAccountName=${STORAGE_ACCOUNT_NAME}" "tfStorageContainerName=terraform-state" \
+			${KV_ARG} \
+			"enableKVPurgeProtection=${KV_PURGE_PROTECTION}" \
+			${WHAT_IF}
 
 deploy-arm-resources: arm-deployment
 
 validate-arm-resources: set-what-if arm-deployment
+
+domains-infra-init: composed-variables domains set-azure-account
```
Comment (saliceti) [inline Makefile:76]: This PR is about postgres. Can we keep the domain commands for later?

File: .gitignore (hunk 1 above)
Comment (temitope777) [reply .gitignore:1]: missing existing ignore entries re-added

File: Makefile (hunk 3 above)
Comment (temitope777) [reply Makefile:10]: done

File: Makefile (hunk 4 above)
Comment (temitope777) [reply Makefile:76]: domains commands removed

File: global_config/production.sh [hunk 5]
```diff
@@ -0,0 +1,5 @@
+CONFIG=production
```
[the remark this answers is not in the document]
Comment (temitope777) [reply global_config/production.sh:1]: references to prod removed

File: Makefile [hunk 6]
```diff
@@ -89,6 +92,12 @@ install-fetch-config:
 	    && chmod +x fetch_config.rb \
 	    || true
 
+install-terrafile: ## Install terrafile to manage terraform modules
```
Comment (saliceti) [inline Makefile:95]: Can you replace this with the simplified version? https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/Makefile#L30

Comment (temitope777) [reply Makefile:95]: removed

**Review by saliceti — APPROVED** [review]

**Pull request merged**